src/apps/content-editor/src/app/views/ItemEdit/FreestyleWrapper.tsx

```diff
@@ -15,6 +19,21 @@ export const FreestyleWrapper = () => {
   // @ts-expect-error CONFIG not typed
   const [sessionToken] = useState(Cookies.get(CONFIG.COOKIE_NAME));
 
+  const [showZestyDAM, setShowZestyDAM] = useState(false);
+
+  const handleZestyDAMRequest = (event: MessageEvent) => {
+    if (event.data.type === "ZESTY_DAM_REQUEST") {
```

Comment:
As I am looking at this I have a thought on how we can further secure our windowing messages. In the vein of what has been discussed in the past with a need to make a robust messaging system as we further expand on embedding external applications. When manager-ui mounts an application it hands that application the current users session token. Additional if a standalone application is running it assumedly used the app loader and obtained a zesty user session token. Either way there should be an active user session token present within both the parent and child application.

If the child application was to provide the session token back to the parent with every window message the parent could then check for a string match between the parents session token and child provided session token. This would confirm that the user session being used between the parent and child is the same.

Thoughts?